File: src/sdk-reference/js/6/realtime/unsubscribe/snippets/unsubscribe.js
function callback (notification) {}

try {
  const { roomId } = await kuzzle.realtime.subscribe('nyc-open-data', 'yellow-taxi', {}, callback);

  await kuzzle.realtime.unsubscribe(roomId);

  console.log('Success');
} catch (error) {
  console.error(error.message);
}

File: src/sdk/java/1/controllers/realtime/unsubscribe/snippets/unsubscribe.java
NotificationListener listener = new NotificationListener() {
  public void onMessage(NotificationResult notification) {}
};

try {
    String roomId = kuzzle.getRealtime().subscribe(
      "nyc-open-data", 
      "yellow-taxi", 
      "{}", 
      listener
    );

    kuzzle.getRealtime().unsubscribe(roomId);

    System.out.println("Success");
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}
